### DISCUSSION TITLE
Draggable minimap mask controls main flow view (like in vs code)

### DISCUSSION THREAD

**Discussion by felixoros** [xyflow/xyflow#1882] [New Features]
I think it would be nice if there will be something like isDraggable prop on the MiniMap component that behaves like in vscode, so the main flow view can be controlled by dragging the mask on the mini map.

**Comment by david-vendel** [comment]
Is there any answer to this? I need the same? 
What is the best solution how to allow this right now? Anything easier than creating custom mini map? Have anyone created custom mini map? 
Thanks

**Comment by vaibhav-systango** [comment]
any solutions for same @justdvl @felixoros

**Comment by david-vendel** [comment]
Hey I found this [Codesandbox](https://codesandbox.io/s/react-flow-interactive-minimap-26u9lj?file=/src/components/MiniMap/index.tsx ) somewhere in discussions here, hope it helps/

**Reply by vaibhav-systango** [reply]
tried this but doesn't seems to work with the current versions of react-flow :(

**Reply by vaibhav-systango** [reply]
@justdvl  do you have it working if yes let me know what version of react-flow you used and share the sample code here if possible ?

**Comment by moklick** [comment]
locked in favour of #2166
